docs/output.md

```diff
@@ -266,10 +301,36 @@ More information on the report can be found in the [pixelator documentation](htt
 - `pipeline_info/`
   - Reports generated by Nextflow: `execution_report.html`, `execution_timeline.html`, `execution_trace.txt` and `pipeline_dag.dot`/`pipeline_dag.svg`.
   - Reports generated by the pipeline: `pipeline_report.html`, `pipeline_report.txt` and `software_versions.yml`. The `pipeline_report*` files will only be present if the `--email` / `--email_on_fail` parameter's are used when running the pipeline.
-  - Reformatted samplesheet files used as input to the pipeline: `samplesheet.valid.csv`.
   - Metadata file with software versions, environment information and pipeline configuration for debugging: `metadata.json`
   - Parameters used by the pipeline run: `params.json`.
 
 </details>
 
 [Nextflow](https://www.nextflow.io/docs/latest/tracing.html) provides excellent functionality for generating various reports relevant to the running and execution of the pipeline. This will allow you to troubleshoot errors with the running of the pipeline, and also provide you with other information such as launch commands, run times and resource usage.
+
+## Output directory structure
+
+With default parameters, the pixelator pipeline output directory will only include the most "complete" PXL file
+generated by the pipeline and an interactive HTML report per sample.
+The PXL dataset files can be from either the `annotate`, `analysis` or `layout` step.
+
+With default parameters, the `<sample-id>.layout.datasets.pxl` will be copied to the output directory.
+If the `layout` stage is skipped (using `--skip_layout`) the `<sample-id>.analysis.datasets.pxl` files will be included and
+if the `analysis` stage is skipped (using `--skip_analysis`) the `<sample-id>.annotate.datasets.pxl` will be copied.
+
+Various flags are available to store intermediate files and are described in the input parameter documentation. Alternatively, you can keep all intermediate files using `--save_all`.
+
+Below is an example output structure for a pipeline run using the default settings.
+
+- `pipeline_info/`
+- `pixelator/`
+
+  - `logs/`
+
+    - `<sample-id>/`:
+      - `*.log`
+
+  - `pbmcs_unstimulated.layout.dataset.pxl`
```

Comment:
Will we keep the `layout`tag on all these samples or that will be stripped?

Reply:
We are publishing the dataset to the outdir with either the annotate, analysis or layout tag present in the filename depending on what the last step of the pipeline was. I think it is important to keep those to be able to quickly see what information will be present in the pxl files.

Reply:
Ok, I agree.